boards/common/stm32/include/l0/cfg_clock_32_16_1.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2018-2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_stm32
+ * @{
+ *
+ * @file
+ * @brief       Configure STM32L0 clock to 32MHz with LSE
```

Comment:
The wording here makes me think 32Mhz are achieved by using the LSE which is incorrect. Maybe reword to
`Configure STM32L0 clock to 32MHz using PLL with LSE`, what do you think?

Reply:
Updated and directly squashed